diem-move/diem-framework/experimental/sources/MultiToken.move

```diff
@@ -89,6 +93,19 @@ module MultiToken {
         Vector::borrow(tokens, index).supply
     }
 
+    spec fun is_in_tokens<TokenType>(tokens: vector<TokenData<TokenType>>, token_id: GUID::ID): bool {
+        exists i in 0..len(tokens): tokens[i].token_id.id == token_id
+    }
+
+    spec supply {
+        let addr = GUID::id_creator_address(id);
+        let token_collection = get_tokens<TokenType>(addr);
+        let min_token_id = choose min i in 0..len(token_collection) where token_collection[i].token_id.id == id;
```

Comment:
Can use `choose min i in range(token_collection) where ...`

In general the source of truth of all current bultin functions is here: https://github.com/diem/diem/blob/main/language/move-model/src/builder/spec_builtins.rs

Reply:
Isn't min_token_id an index and not a (GU)ID?